### DISCUSSION TITLE
Run the Simulation with the trained Model

### DISCUSSION THREAD

**Discussion by diksanm7** [gymprecice/gymprecice#6] [Q&A] [answered]
Hi, I am new to this RL things. If I run the ppo_controller.py (I am using jet cylinder case for now), is it going to train the model right? So, after the model trained and there will be .pt file / the saved / the trained model, how to run the simulation again with the trained model?

**Comment by mosayebshams** [comment, marked as answer]
Hi, please take a look [here](https://github.com/gymprecice/tutorials/blob/main/quickstart/closed_loop_ppo_controller.py) to see how to load and utilise a trained agent as a controller.
